### PR TITLE
Pooling min/max

### PR DESCRIPTION
**Pull request by naveen-venkat**
Type casting the min max values to integer

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

File: lib/ibmdb.js [hunk 1]
```diff
@@ -52,9 +52,9 @@ function IBMDB(name, settings) {
 
   // Save off the connectionOptions passed in for connection pooling
   this.connectionOptions = {};
-  this.connectionOptions.minPoolSize = (settings.minPoolSize || 0);
-  this.connectionOptions.maxPoolSize = (settings.maxPoolSize || 0);
-  this.connectionOptions.connectionTimeout = (settings.connectionTimeout || 60);
+  this.connectionOptions.minPoolSize = parseInt((settings.minPoolSize || 0));
+  this.connectionOptions.maxPoolSize = parseInt((settings.maxPoolSize || 0));
+  this.connectionOptions.connectionTimeout = parseInt((settings.connectionTimeout || 60));
```
Comment (superkhau) [inline lib/ibmdb.js]:
```
this.connectionOptions.connectionTimeout = parseInt(settings.connectionTimeout) || 60;
```

For all the above.

**Comment by superkhau** [comment]
@naveen-venkat Thanks for the contribution! Will take it over it over to help get it landed since this PR is out of date.

**Comment by superkhau** [comment]
@naveen-venkat Helped you land this PR via #32, thanks for the contribution! :shipit:

**Comment by naveen-venkat** [comment]
thanks!!

**Pull request closed without merging**